### ISSUE TITLE
Pubsub: AttributeError: 'module' object has no attribute 'IAMPolicyStub'

### ISSUE THREAD

**Issue by TrangPham** [googleapis/google-cloud-python#7197]
I am filing this bug report with my solution which worked.

Issue seems to stem from this line: https://github.com/googleapis/google-cloud-python/blob/ca33b0d015ab7956b881b6964647f28f9f999f0f/pubsub/google/cloud/pubsub_v1/gapic/transports/publisher_grpc_transport.py#L71

#### Environment details

google-cloud-pubsub          0.39.1             
OS: debian
Python 2.7
```
root@cbb6530414df:/app# pip list  | grep proto
googleapis-common-protos (1.6.0b6)
protobuf (3.6.1)
root@cbb6530414df:/app# pip list | grep google
google-api-core (1.7.0)
google-auth (1.6.2)
google-cloud-pubsub (0.39.1)
googleapis-common-protos (1.6.0b6)
googlemaps (2.4.5)
grpc-google-iam-v1 (0.11.4)
```

#### Code example

```python
from google.cloud import pubsub_v1

subscriber = pubsub_v1.SubscriberClient()
```

#### My working solution

I was getting this error on my docker environment but not locally. I compared my pip versions and noticed `googleapis-common-protos (1.5.6)` locally. Looking at the `iam_policy_pb2.py` file said it was generated from proto.

The solution which worked was setting `googleapis-common-protos==1.5.6` in my docker's requirement.txt file.

**Comment by cs82** [comment]
I have the same problem, but sadly the solution doesn't seem to work for me.

Environment:
Debian 8
Python 3.4.2
google-cloud-pubsub 0.39.1

Module List:
```
#pip3 list | grep google
google-api-core (1.7.0)
google-api-python-client (1.7.8)
google-auth (1.6.2)
google-auth-httplib2 (0.0.3)
google-cloud-pubsub (0.39.1)
googleapis-common-protos (1.6.0b6)
grpc-google-iam-v1 (0.11.4)
```

With version 1.6.0b6 of googleapis-common-protos I get the error
`AttributeError: 'module' object has no attribute 'IAMPolicyStub'`

I set up a virtualenv with version 1.5.6 of googleapis-common-protos, but then I get the error

```
Traceback (most recent call last):
  File "createSubscription.py", line 12, in <module>
    from google.cloud import pubsub_v1
  File "/pubsub/lib/python3.4/site-packages/google/cloud/pubsub_v1/__init__.py", line 17, in <module>
    from google.cloud.pubsub_v1 import types
  File "/pubsub/lib/python3.4/site-packages/google/cloud/pubsub_v1/types.py", line 20, in <module>
    from google.iam.v1 import iam_policy_pb2
ImportError: cannot import name 'iam_policy_pb2'

```

**Comment by tseaver** [comment]
`googleapis-common-protos 1.6.0` was released today:  upgrading from `1.6.0b#` should resolve this issue.
